recipes/nexpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "nexpy" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/NeXpy-{{ version }}.tar.gz"
+  sha256: "c8b6080668727ebb131751924e7ab5418bc451a415f616a293b037e0a0778dcf"
+
+build:
+  number: 0
+  entry_points:
+    - nexpy = nexpy.nexpygui:main
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
is setuptools and numpy rally needed as host dependency?

Reply:
Yes, setuptools is definitely required.

Numpy shouldn't be, but it is imported in setup.py. I fixed this yesterday in master with https://github.com/nexpy/nexpy/pull/201